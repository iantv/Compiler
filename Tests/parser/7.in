int func(){
	int a, b;
	{
		double c;
	}
}