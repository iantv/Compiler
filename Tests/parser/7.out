b: variable
	int
func: function
	a: variable
		int
	block:
		c: variable
			double
returns
	int
